packages/bitwarden/data_stream/member/fields/fields.yml

```
@@ -0,0 +1,66 @@
+- name: bitwarden
+  type: group
+  fields:
+    - name: member
+      type: group
+      fields:
+        - name: access_all
+          type: boolean
+          description: Determines if this member can access all collections within the organization, or only the associated collections. If set to {true}, this option overrides any collection assignments.
```

```suggestion
          description: Determines if this member can access all collections within the organization, or only the associated collections. If set to true, this option overrides any collection assignments.
```
or
```suggestion
          description: Determines if this member can access all collections within the organization, or only the associated collections. If set to `true`, this option overrides any collection assignments.
```
Its unclear to me why they used curlies for this in the swagger definition; it's not a parameter and the description is not a parameterisable entity.

---

Will Update it.